controller/site_config/mac_addrs/9a02571e8f03/device_config.json

```diff
@@ -1,3 +1,8 @@
 {
-    "static_ip": "192.168.1.3"
+    "static_ip": "192.168.1.3",
+    "modules": {
+        "hold": {
+            "enabled": true
```

Comment:
e.g., even with "test_hold" flag not set, I would argue that this setting here should invoke the hold test just for this device.

Reply:
It does but I wanted to make it explicit so if the base config changes, hold will still be enabled.

Reply:
It doesn't since the hold test is not part of the "configured tests". That's why I really wanted to remove the host_tests concept so only the module_config configures for what modules to run.

Reply:
I verified with 1.10.5 and these configs that hold test doesn't run unless you specify the -k. Hold test isn't part of the configured tests so enabling it in config alone doesn't run the test.